docs/proposals/docker-registry.md

```diff
@@ -0,0 +1,33 @@
+## Improve docker-registry
+
+We need to stop using `linkedBlobStore` from `docker/distribution`. The main purpose for this module is to store information
+about blobs because `docker/distribution` don't have own database. It places information in the filesystem.
+
+Openshift already uses an etcd as a database. All infomation about images/imagestreams are there. We can hold information
+about the blobs in repository in the database as well. As result, we need to store only the blobs's payload in the filesystem
```

Comment:
Please detail where and how this information would be stored in etcd.  On the existing imagestream object, a new meta object, etc?  Also please detail what pieces would need to change to update this information.